_meta/config/cloudbeat.common.yml.tmpl

```diff
@@ -1,7 +1,12 @@
 cloudbeat:
   # Defines how often an event is sent to the output
   period: 30s
-  files: [
+  fetchers:
```

Comment:
Nice!
The file-system fetcher config struct seem to match this already but it needs to be validated somehow (manual test/ the better solution will be a simple test on `parseConfigFetchers` with the default configuration and a validation of the created parsed fetchers and their config).